aptos-move/framework/aptos-framework/doc/transaction_fee.md

```diff
@@ -805,7 +805,7 @@ Only called during genesis.
 <b>requires</b> <b>exists</b>&lt;<a href="transaction_fee.md#0x1_transaction_fee_AptosCoinCapabilities">AptosCoinCapabilities</a>&gt;(@aptos_framework);
```

Comment:
The linter fails in CI due to the outdated md files: https://github.com/aptos-labs/aptos-core/actions/runs/6580606157/job/17878952753?pr=10349
Please do `cargo build -p aptos-cached-packages` to regenerate the md files before (force-)pushing the commit.